### DISCUSSION TITLE
Do you have any plan to release a new version?

### DISCUSSION THREAD

**Discussion by bear0330** [getredash/redash#6614] [General]
Hi, the last release is two years ago. I want to try redash but I don't know where to get the latest stable release not two years ago version.
Do you have any plan to release a new version? Redash is great.
Thanks.

**Comment by guidopetri** [comment]
Hi! We're planning on figuring out how to make a formal release at our next meeting this weekend. In the meantime, you can use the `preview` image tag on Dockerhub, but please keep in mind that not all functionality may be working.
